XamlControlsGallery/ControlPages/SliderPage.xaml

```diff
@@ -27,21 +28,65 @@
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+        
         <local:ControlExample x:Name="Example2" HeaderText="A Slider with range and steps specified.">
-            <StackPanel x:Name="Control2" Orientation="Horizontal">
-                <Slider x:Name="Slider2" Header="Control header" Width="200" Minimum="500" Maximum="1000"
-                        LargeChange="100" StepFrequency="10" SmallChange="10" Value="800"/>
-            </StackPanel>
+            <local:ControlExample.Options>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="Auto"/>
```

Comment:
Nit: Set the `MinWidth` property so that the column doesn't resize upon changes that go from 3 digit to 4 digit numbers.